public/sass/components/_slate_editor.scss

```diff
@@ -35,7 +35,6 @@
     border: $panel-border;
     max-height: calc(66vh);
     overflow-y: scroll;
-    max-width: calc(66%);
```

Comment:
If this was left in, the typeahead box renders slimmer than normal, until an item is selected. No idea why.